test/get-predeployed-accounts.test.ts

```diff
@@ -13,4 +14,16 @@ describe("Devnet restart", function () {
         expect(response[0].private_key).to.be.a("string");
         expect(response[0].public_key).to.be.a("string");
     });
+
+    it("should fetch n number of OZ account classes from predeployed accounts", async () => {
```

Comment:
What's the point in doing the same thing twice in a row? Maybe we don't need the new utility function, especially if this getting of OZ accounts is just going to be done once.

Reply:
I wrote two parts of `getPredeployedOZAccounts()` because it tests the newly introduced `ensurePort()` function. If we remove `ensurePort()` and just do `getFreePort()`, a single call to `getPredeployedOZAccounts()` will work but if u do multiple calls, it'll fail.